mix.exs

```diff
@@ -27,6 +27,7 @@ defmodule Postgrex.Mixfile do
     [{:ex_doc, only: :dev},
      {:earmark, only: :dev},
      {:decimal, "~> 1.0"},
+     {:db_connection, "~> 0.1.6"},
      {:connection, "~> 1.0"}]
```

Comment:
We can't, its used for notification process

On Monday, 4 January 2016, José Valim <notifications@github.com> wrote:

> In mix.exs
> <https://github.com/ericmj/postgrex/pull/128#discussion_r48724037>:
>
> > @@ -27,6 +27,7 @@ defmodule Postgrex.Mixfile do
> >      [{:ex_doc, only: :dev},
> >       {:earmark, only: :dev},
> >       {:decimal, "~> 1.0"},
> > +     {:db_connection, "~> 0.1.6"},
> >       {:connection, "~> 1.0"}]
>
> Should we remove this dep?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ericmj/postgrex/pull/128/files#r48724037>.
>